src/test/resources/soop_sst_trajectory.xml

```diff
@@ -6,68 +6,849 @@
+    <attribute name="featureType" value="'trajectory'"/>
+    <attribute name="naming_authority" value="'IMOS'"/>
+    <attribute name="history" sql="select history from $instance"/>  --????
+    <attribute name="geospatial_lat_min">
+      <sql>select min("LATITUDE") from $instance</sql>
+    </attribute>
+    <attribute name="geospatial_lat_max">
+      <sql>select max("LATITUDE") from $instance</sql>
+    </attribute>
+    <attribute name="geospatial_lon_min">
+      <sql>select min("LONGITUDE") from $instance</sql>
+    </attribute>
+    <attribute name="geospatial_lon_max">
+      <sql>select max("LONGITUDE") from $instance</sql>
+    </attribute>
+    <attribute name="geospatial_vertical_min" value="0"/>
+    <attribute name="geospatial_vertical_max" value="0"/>
+    <attribute name="time_coverage_start">
+      <sql>select to_char(min("TIME" AT TIME ZONE 'UTC'), 'YYYY-MM-DD"T"HH24:MI:SS"Z"') from $data</sql>
+    </attribute>
+    <attribute name="time_coverage_end">
+      <sql>select to_char(max("TIME" AT TIME ZONE 'UTC'), 'YYYY-MM-DD"T"HH24:MI:SS"Z"') from $data</sql>
+    </attribute>
+    <attribute name="data_centre" value="'eMarine Information Infrastructure (eMII)'"/>
+    <attribute name="data_centre_email" value="'info@emii.org.au'"/>
+    <attribute name="institution_references" value="'http://www.imos.org.au/emii.html'"/>
 
+    <attribute name="citation">
+      <value>'The citation in a list of references is: "IMOS [year-of-data-download], [Title], [data-access-url], accessed [date-of-access]"'</value>
+    </attribute>
+    <attribute name="acknowledgement">
+      <value>'Any users of IMOS data are required to clearly acknowledge the source of the material in the format: "Data was sourced from the Integrated Marine Observing System (IMOS) - an initiative of the Australian Government being conducted as part of the National Collaborative Research Infrastructure Strategy and and the Super Science Initiative."'</value>
+    </attribute>
+    <attribute name="distribution_statement">
+      <value>'Data may be re-used, provided that related metadata explaining the data has been reviewed by the user, and the data is appropriately acknowledged. Data, products and services from IMOS are provided "as is" without any warranty as to fitness for a particular purpose.'</value>
+    </attribute>
+    <attribute name="project_acknowledgement">
+      <value>'The collection of this data was funded by IMOS'</value>
+    </attribute>
+  </globalattributes>
+  <variables>
     <variable>
-      <name>id</name>
+      <name>trajectory</name>
       <encoder>integer</encoder>
       <attributes>
-        <attribute name="_FillValue" value="99999"/>
+        <attribute name="long_name" value="Unique identifier for each feature instance"/>
+        <attribute name="cf_role" value="trajectory_id"/>
       </attributes>
     </variable>
-
     <variable>
       <name>TIME</name>
-      <encoder>time</encoder>
+      <encoder>time</encoder>  
```

Comment:
@julian1 : Bene says "i am  following what's in the anmn_ts template but questioning the encoder that is used here 'time'. NODC recommends time to be of type double, which is also the current IMOS convention. Why using type 'time' then (except the fact that TIME  is a timestamp in the DB). TIME should be converted back to double to conform the NODC format? Am i missing something here?"

Good point, but I think TIME is output as a Double anyway. Julian might have a better explanation of why he set TIME encoder to time.